configs/prometheus-rds-exporter/prometheus-rds-exporter.yaml

```diff
@@ -1,17 +1,47 @@
-# Log format (text or json)
-# log-format: json
+#
+# Exporter settings
+#
+
+# Enable debug mode
+#debug: false
 
 # Path under which to expose metrics
-# metrics-path: /metrics
+#metrics-path: /metrics
 
 # Address to listen on for web interface
-# listen-address: ":9043"
+#listen-address: ":9043"
+
+# Log format (text or json)
+#log-format: json
 
-# AWS profile
-# aws-profile: my_profile
+#
+# AWS credentials
+#
 
 # AWS IAM ARN role to assume to fetch metrics
-# aws-assume-role-arn: arn:aws:iam::000000000000:role/prometheus-rds-exporter
+#aws-assume-role-arn: arn:aws:iam::000000000000:role/prometheus-rds-exporter
 
 # AWS assume role session name
-# aws-assume-role-session: prometheus-rds-exporter
+#aws-assume-role-session: prometheus-rds-exporter
+
+#
+# Metrics
+#
+
+# Collect AWS instances metrics (AWS Cloudwatch API)
+#collect-instance-metrics: true
+
+# Collect AWS instance types information (AWS EC2 API)
+#collect-instance-types: true
+
+# Collect AWS instances logs size (AWS RDS API)
+#collect-logs-size: true
+
+# Collect AWS instances maintenances (AWS RDS API)
+#collect-maintenances: true
+
+#Collect AWS RDS quotas (AWS quotas API)
+#collect-quotas: true
+
+# Collect AWS RDS usages (AWS Cloudwatch API)
+#collect-usages: true
```

Comment:
nitpick: While I think I understand why you separated comments from actual parameters by distinguishing them with a space, if you used

```yaml
# # Collect AWS RDS usages (AWS Cloudwatch API)
# collect-usages: true
``` 
or

```yaml
## Collect AWS RDS usages (AWS Cloudwatch API)
# collect-usages: true
```
Someone using VScode may simply use the `Command⌘` + `K` + `U` shortcut to uncomment all parameters from the file, and it's not too ugly for VI users.

Reply:
It's a tradeoff between YAML valid syntax and user experience: `#collect-usages: true` is not YAML-lint valid (not yet in the project) and `# # Collect AWS RDS usages (AWS Cloudwatch API)` is ugly :(

Let's discuss this YAML standard in another PR.

Reply:
For visibility, we are introducing YAML linter in https://github.com/qonto/prometheus-rds-exporter/pull/31